legal-api/src/legal_api/models/business.py

```diff
@@ -265,10 +262,7 @@ class AssociationTypes(Enum):
 
     jurisdiction = db.Column('foreign_jurisdiction', db.String(10))
     foreign_jurisdiction_region = db.Column('foreign_jurisdiction_region', db.String(10))
```

Comment:
Used for continuation out

Reply:
Good catch. Yes, you need to change that check once this code is merged

Reply:
@chenhongjing this is different from involuntary dissolution check. You are checking an expro business whereas this creates a continuation in business (C, CBEN, CCC, CUL). But when we implement expro filing we have to update that check according to the table structure